RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# Releasing Percona Migrator
+
+All releases come from the master branch. Only those releases that add a fix for
+the Rails 3.2 support will come from the branch v3.2. Keep in mind though that
```

Comment:
maybe reword to: `all branches except master considered deprecated and will receive only bug fix releases`?